### PR TITLE
task4 commit

### PR DESCRIPTION
**Pull request by impenetrabl1**
PR feature4 - develop

### CODE REVIEW

File: task4.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import calendar
+
```
Comment (MaksimLion) [inline task4.py]: the function must be called by meaning

File: task4.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+def func4(year):
+    if year % 4 != 0:
+        print('обычный')
+    elif year % 100 != 0:
+        print('высокосный')
+    elif year % 400 != 0:
+        print('обычный')
+    else:
+        print('высокосный')
+
```
Comment (MaksimLion) [inline task4.py]: think about how you can join at one condition

File: task4.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+def func4(year):
+    if year % 4 != 0:
+        print('обычный')
+    elif year % 100 != 0:
+        print('высокосный')
+    elif year % 400 != 0:
+        print('обычный')
+    else:
+        print('высокосный')
```
Comment (MaksimLion) [inline task4.py]: function must return instead print

File: task4.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+
+def func4_2(year):
+    if calendar.isleap(year):
+        print('высокосный')
+    else:
+        print('обычный')
+
```
Comment (MaksimLion) [inline task4.py]: replace print and extra block else

File: new_task4.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+
+def is_leap_year(year_param):
+    if (year_param % 4 == 0) and (year_param % 100 != 0) or (year_param % 400 == 0):
+        return True
+    return False
+
+
+def is_leap_year_2(year_param):
+    if calendar.isleap(year_param):
```
Comment (AlehSivakou) [inline new_task4.py]: Unnecessary return True/return False block. You can return exact condition value in one line of code.

File: new_task4.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    return (year_param % 4 == 0) and (year_param % 100 != 0) or (year_param % 400 == 0)
+
+
+def is_leap_year_2(year_param):
+    return calendar.isleap(year_param)
+
+
+with FileOpen(r'C:\Users\Aliaksandr.Baryhin\Desktop\dataFiles\task4_data.txt', 'a') as open_file:
+    open_file.write('\nIs this year is leap? ' + str(is_leap_year(year)))
+    open_file.write('\nIs this year is leap? ' + str(is_leap_year_2(year)))
```
Comment (MaksimLion) [inline new_task4.py]: you can use "{}".format() instead concatenation string

**Pull request merged**